site/app/libraries/database/DatabaseQueries.php

```diff
@@ -724,13 +724,17 @@ public function getGradeableVersionHasAutogradingResults($g_id, $version, $user_
         return count($this->course_db->rows()) > 0 && $this->course_db->rows()[0]['autograding_complete'] === true;
     }
 
+    protected function constructParenthesizedQuestionMarksByLength($len) {
```

Comment:
Let's come up with a better name as this is longer than the line of code it's replacing. `createParamaterList`?